.github/workflows/release-please.yml

```
@@ -35,6 +35,8 @@ jobs:
           node-version: lts/*
           registry-url: https://registry.npmjs.org
         if: ${{ steps.release.outputs.release_created }}
+      - run: npm install
+        if: ${{ steps.release.outputs.release_created }}
       - run: npm publish
```

I think you can simplify this by using heredocs:
```suggestion
      - run: |
          npm install
          npm publish
```